builds/misc/addons-release.yaml

```diff
@@ -1,12 +1,18 @@
 name: $(version)
-jobs: 
+jobs:
 ################################################################################
   - job: linux_API_proxy_module
 ################################################################################
     displayName: linuxAPIProxy
     pool:
       vmImage: 'ubuntu-18.04'
     steps:
+      - bash: |
+            build_version='$(version)'
+            if [ -z "$build_version" ] then
+              echo '##vso[task.setvariable variable=buildVersion]$(Build.BuildNumber)'
```

Comment:
`buildVersion` needs to be set to `$(version)` in the else case.

Reply:
Just realized that :D
Fixed